packages/create-bison-app/template/pages/index.tsx

```diff
@@ -17,10 +20,28 @@ function Home() {
       </Head>
 
       <Center>
-        <Heading size="lg">{welcomeMsg}</Heading>
+        <Heading size="lg">{welcomeMessage}</Heading>
       </Center>
     </>
   );
 }
 
 export default Home;
+
+export const getServerSideProps: GetServerSideProps<PageProps> = async (context: any) => {
```

Comment:
Is it throwing an error when removing `any` from `context`? I thought it was inferred from `GetServerSideProps`